apps/data/src/lib/data_repair/data_repair/manifest.py

```
@@ -30,11 +35,20 @@ def inline_repair(
+    runs against ``table``.
+
+    The decorated function receives the DataFrame plus a ``gate`` keyword
+    argument: a Column<Boolean> derived from ``predicate`` (or always-True if
+    no predicate). The repair AND's its own conditions (content signature,
+    schema-shape checks) into ``gate`` before applying the inverse.
+
+    severity="advisory" registers without applying.
+    """
 
-    def _decorate(fn: Callable[[DataFrame], DataFrame]) -> Callable[[DataFrame], DataFrame]:
+    def _decorate(fn: Callable[..., "DataFrame"]) -> Callable[..., "DataFrame"]:
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)